src/viewProviders/BaseWebviewViewProvider.ts

```diff
@@ -263,7 +263,7 @@ export abstract class BaseWebviewViewProvider implements vscode.WebviewViewProvi
             enableScripts: true,
 
             localResourceRoots: [
-                vscode.Uri.file(this.webviewBasePath)
+                vscode.Uri.file('/')
```

Comment:
I don't think this change is correct. webviewBasePath is set to `path.join(extensionContext.extensionPath, 'dist', 'webviews');`